magento/inventory
# Msi 2182 source entity expansion with pickup location attributes

### Description (*)
Added the next fields:
- Frontend Name
- Frontend Description (wysiwyg field, label is not shown in BO)
- Implemented admin UI for them;
- Refactored some code;

### Fixed Issues (if relevant)
1. magento-engcom/msi#2182:Source Entity expansion with Pickup Location attributes

### Manual testing scenarios (*)
1. Go to source edit page
2. Make sure pickup location fieldset is not visible until the source is used in the store pickup
3. Check that new fields are present
4. Make sure you are able to save them 

### Contribution checklist (*)
